### PR TITLE
Add additional effect mods for Tigerhunter

### PR DESCRIPTION
**Pull request by Grahf0085**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x ] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

**Review by TeoTwawki — DISMISSED** [review]
thats not how these are handled anymore. To add additional effects you just add item mods now.

edit:
github being trash, disregard the "approval". I most definitely do not approve.

**Review by TeoTwawki — CHANGES_REQUESTED** [review]
~~see existing examples in item mods~~
Here is how it would look:
```sql
-- Tigerhunter 
INSERT INTO `item_mods` VALUES (16672,431,2);  -- Additional effect type: DEBUFF (PARALYSIS)
INSERT INTO `item_mods` VALUES (16672,499,11); -- Additional effect animation (subEffect) xi.subEffect.PARALYSIS
INSERT INTO `item_mods` VALUES (16672,501,10); -- Additional effect Chance 10%
```
Assuming the rate is really 10% that is

File: sql/item_mods.sql [hunk 1]
```diff
@@ -20329,6 +20329,9 @@ INSERT INTO `item_mods` VALUES (16660,11,3);
 INSERT INTO `item_mods` VALUES (16662,9,4);
 INSERT INTO `item_mods` VALUES (16662,11,4);
 INSERT INTO `item_mods` VALUES (16669,23,5);
+INSERT INTO item_mods VALUES (16672,431,2);  -- Additional effect type: DEBUFF (PARALYSIS)
```
Comment (TeoTwawki) [inline sql/item_mods.sql:20332]: you left out the item name

Comment (TeoTwawki) [reply sql/item_mods.sql:20332]:
```sql
INSERT INTO `item_mods` VALUES (16669,23,5);

-- Tigerhunter
INSERT INTO item_mods VALUES (16672,431,2);  -- Additional effect type: DEBUFF (PARALYSIS)
```

**Review by claywar — COMMENTED** [review]
This weapon should be additional effect paralysis vs tigers only. Is this handled?

**Comment by TeoTwawki** [comment]
> This weapon should be additional effect paralysis vs tigers only. Is this handled?

@Grahf0085 you'll need to use latent effect table for that sort of case Grahf. 

```cpp
    VS_ECOSYSTEM         = 59, // Vs. Specific Ecosystem ID (e.g. Vs. Birds: Accuracy+3)
    VS_FAMILY            = 60, // Vs. Specific Family ID (e.g. Vs. Apkallu: Accuracy+3)
```
I believe you want latent ID 59

**Comment by Grahf0085** [comment]
> 
In my lua I have 
` if (target:getFamily() == 242) then`
Left that line out of the lua when I was translating to LSB format.
I'll look into adding it to the SQL. Thank you

**Comment by claywar** [comment]
Closing this PR for now.  If updates are made, please re-open and we'll be happy to review!

**Pull request closed without merging**